ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -156,14 +164,16 @@ message Right {
         CanActAs can_act_as = 2;
         // The user can read ledger data visible to a specific party.
         CanReadAs can_read_as = 3;
+        // The user can administrate the identity provider.
```

Comment:
Which identity provider will the user be able to administrate?

Reply:
I have added a clarification, please have a look